### PR TITLE
Signal

### LINKED ISSUE #1363: NEW: signal component
A simple component for the "Hotness Signal" in Skin.

https://github.com/eBay/skin/issues/1367

### PR DESCRIPTION
**Pull request by austingardner**
## Description
Signals are data-backed recommendations to help customers make more informed decisions.
There are four signal types, each corresponding to a specific color: trustworthy, recent, time-sensitive & neutral.

## Context
Finishes up the Signal component already merged in Skin. 

## References
Closes #1363 

## Screenshots
Screen Shot 2021-04-09 at 7 48 31 AM

### CODE REVIEW

File: src/components/ebay-signal/index.marko [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$ var type = input.type;
```
Comment (agliga) [inline src/components/ebay-signal/index.marko:1]: We should have a default value
Also I think we generally don't like `type`. Maybe `status` is better?

File: src/components/ebay-signal/index.marko [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+<span class=`signal signal--${input.type}`>
```
Comment (agliga) [inline src/components/ebay-signal/index.marko:3]: Pass in the `...input`

Also do we need any `role` or aria attributes here? At least `role` would probably be needed.

**Review by agliga — COMMENTED** [review]
Can you also add some server tests, just to check that the type is getting set correctly?

File: src/components/ebay-signal/index.marko (hunk 2 above)
Comment (ianmcburnie) [reply src/components/ebay-signal/index.marko:3]: Good point. I'll have a think if we need anything and add it to the Skin docs if so.

**Review by ianmcburnie — APPROVED** [review]

**Pull request merged**